src/platforms/common/data-management/debug-files.mdx

```diff
@@ -593,6 +594,34 @@ Examples:
 - `b5/381a457906d279073822a5ceb24c4bfef94ddb` (executable or library)
 - `b5/381a457906d279073822a5ceb24c4bfef94ddb.debug` (stripped debug file)
 
+**Unified**
+
+: Path: `nn/nnnnnnnnnnnnnnnn.../type`
+
+The unified path layout is a layout specific to Sentry which enables a consistent
+structure for all debug files independent of platform.  It can store breakpad files,
+PDBs, PEs and everything else.  The [symsorter tool](https://github.com/getsentry/symbolicator/tree/master/symsorter)
+can automatically sort debug symbols into this format and also automatically
+create source bundles.
```

Comment:
```suggestion
The unified path layout is specific to Sentry and enables a consistent
structure for all debug files independent of platform.  It can store breakpad files,
PDBs, PEs, and everything else.  The [symsorter tool](https://github.com/getsentry/symbolicator/tree/master/symsorter)
can automatically sort debug symbols into this format and also automatically
create source bundles.
```